envs/monkey_zoo/blackbox/utils/bb_singleton.py

```diff
@@ -0,0 +1 @@
+start_time = 0
```

Comment:
Not happy about this "singleton", but didn't want to invest time into a more robust solution just to trim logs

Reply:
A lot of this test suite needs to be refactored. We can save this for another day.